nelmio/api-doc-bundle/2.13/etc/packages/nelmio_api_doc_routing.yaml

```diff
@@ -0,0 +1 @@
+nelmio_api_doc: ~
```

Comment:
@Pierstoval I approve of enriching this point, but it requires maintenance on this repository if there are changes on the reference configurations of the project.
The question is: is it not wiser to comment only on the link to the documentation?